war/src/main/webapp/css/style.css

```diff
@@ -1891,3 +1891,28 @@ body.no-sticker #bottom-sticker {
+
+.remodal {
```

Comment:
you mean to add new classes to use bem selectors? Or to rename existing ones? Since it is build both on remodal and bootstrap libs there are many existing css rules.